ggml/src/ggml-vulkan/vulkan-shaders/argmax.comp

```diff
@@ -19,19 +21,26 @@ void main() {
     const uint row = gl_WorkGroupID.z * 262144 + gl_WorkGroupID.y * 512 + gl_WorkGroupID.x;
     const uint col = gl_LocalInvocationID.x;
 
-    if (col >= p.KX) {
+    if (row >= p.KY) {
```

Comment:
You're right that the col check did not make sense there, but the row check does not make sense either. The number of workgroups can be set to exactly the right value.

Reply:
Oh yeah, I totally forgot about that. A shame AMD and Intel don't support large x-dimension on the workgroup like Nvidia does.